### DISCUSSION TITLE
Friendly class names for dev env

### DISCUSSION THREAD

**Discussion by alex-kim-dev** [tldraw/tldraw-v1#553] [Ideas]
Hi! I wanted to add this feature to `@tldraw/tldraw` but it appears the dev tooling is in the closed source `@tldraw/lfg` package. Is it possible to add this?

**Comment by judicaelandria** [comment]
can you elaborate what you need here? thanks

**Reply by alex-kim-dev** [reply]
I mean that the dev class names could contain the class names from the css files or the corresponding component's name (for css-in-js).

**Reply by steveruizok** [reply]
I should open up lfg, it's really just a little cli around esbuild. I'm not exactly sure what you mean though: the classnames are created by stitches, which is not ideal. We're using "regular css" in the UI for the next version, so overrides etc should be much easier.

**Reply by alex-kim-dev** [reply]
Sorry for my english, I'll explain by example: instead of `class="c-hinyfY"` in a browser's devtools I wanted to see `class="ComponentOrClassName-hinyfY"`
